week_03/README.md

```diff
@@ -1 +1,35 @@
- 
\ No newline at end of file
+# Use library(ggplot2); library(gapminder); library(dplyr) to load the necessary packages.
```

Comment:
You should put code in a file with the `.R` extension, not `.md`.  Otherwise this looks good.